app-header/app-header.html

```diff
@@ -95,6 +95,9 @@
       position: relative;
       display: block;
       -webkit-transform: translateZ(0);
+      transition-property: -webkit-transform;
+      transition-property: transform;
```

Comment:
@frankiefu We do a similar thing in app-drawer https://github.com/PolymerLabs/app-layout/blob/master/app-drawer/app-drawer.html#L92